src/components/TimeseriesSearchAndSelect/TimeseriesSearchAndSelect.test.tsx

```diff
@@ -0,0 +1,380 @@
+import React from 'react';
+import { mount, configure } from 'enzyme';
+import { Input } from 'antd';
+import lodash from 'lodash';
+import * as sdk from '@cognite/sdk';
+import TimeserieSearchAndSelect from './TimeseriesSearchAndSelect';
+import DetailCheckbox from 'components/DetailCheckbox/DetailCheckbox';
+import { assetsList } from 'mocks/assetsList';
+import { timeseriesList } from 'mocks/timeseriesList';
+import Adapter from 'enzyme-adapter-react-16';
+import Mock = jest.Mock;
+
+configure({ adapter: new Adapter() });
+
+sdk.TimeSeries.search = jest.fn();
+sdk.Assets.list = jest.fn();
+
+const { Search } = Input;
+const propsCallbacks: { [name: string]: Mock } = {
```

Comment:
Do you need to specify the type here? I would just remove `: { [name: string]: Mock }`

Reply:
Done